packages/office-ui-fabric-react/src/utilities/positioning/positioning.types.ts

```diff
@@ -100,18 +100,6 @@ export interface IPosition {
   [key: string]: number | undefined;
 }
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-export interface Point {
```

Comment:
This should probably re-export Point and IPoint from utilities for now

Reply:
fixed.